roles/openshift_cluster_metrics_hawkular/vars/main.yaml

```diff
@@ -0,0 +1,17 @@
+hawkular_tasks:
+    - policy add-role-to-user edit system:serviceaccount:openshift-infra:metrics-deployer -n openshift-infra
+    - policy add-cluster-role-to-user cluster-admin system:serviceaccount:openshift-infra:heapster
+
+deployer_service_account:
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: metrics-deployer
+    secrets:
+    - name: metrics-deployer
+
+hawkular_tmp_conf: /tmp/hawkular_admin.kubeconfig
+
+hawkular_hostname: {{ openshift.master.metrics_public_url | default("hawkular-metrics.example.com") }} 
```

Comment:
Sorry, I was unclear, I meant to add outer double quotes around the {{...}}:
```
hawkular_hostname: "{{ openshift.master.metrics_public_url | default('hawkular-metrics.example.com') }}"
```
the change to single quote inside was just to not require escaping.
With your a9a6ba commit it still says:
```
ERROR: Syntax Error while loading YAML script, /home/bpaskinc/openshift-ansible/playbooks/common/openshift-cluster/roles/openshift_cluster_metrics_hawkular/vars/main.yaml
Note: The error may actually appear before this position: line 15, column 21


hawkular_hostname: {{ openshift.master.metrics_public_url | default('hawkular-metrics.example.com') }} 
                    ^
We could be wrong, but this one looks like it might be an issue with
missing quotes.  Always quote template expression brackets when they
start a value. For instance:

    with_items:
      - {{ foo }}

Should be written as:

    with_items:
      - "{{ foo }}"
```